### PR TITLE
Add Torch TD3 and DQN to README

### PR DESCRIPTION
**Pull request by maliesa96**
@irisliucy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,13 +56,13 @@ The table below summarizes the algorithms available in garage.
 | CMA-ES                 | numpy               |
 | REINFORCE (a.k.a. VPG) | PyTorch, TensorFlow |
 | DDPG                   | PyTorch, TensorFlow |
-| DQN                    | TensorFlow          |
+| DQN                    | Pytorch, TensorFlow |
```
Comment (ryanjulian) [inline README.md:59]:
```suggestion
| DQN                    | PyTorch, TensorFlow |
```

File: README.md [hunk 2]
```diff
@@ -56,13 +56,13 @@ The table below summarizes the algorithms available in garage.
...
 | DDPG                   | PyTorch, TensorFlow |
-| DQN                    | TensorFlow          |
+| DQN                    | PyTorch, TensorFlow |
 | DDQN                   | TensorFlow          |
 | ERWR                   | TensorFlow          |
 | NPO                    | TensorFlow          |
 | PPO                    | PyTorch, TensorFlow |
 | REPS                   | TensorFlow          |
-| TD3                    | TensorFlow          |
+| TD3                    | Pytorch, TensorFlow |
```
Comment (ryanjulian) [inline README.md:65]:
```suggestion
| TD3                    | PyTorch, TensorFlow |
```

**Review by ryanjulian — APPROVED** [review]
See my suggets

**Review by yeukfu — APPROVED** [review]

**Comment by maliesa96** [comment]
quickest PR in history

**Pull request merged**